README.md

```diff
@@ -1 +1,46 @@
-# demand-capacity-mgmt
+# Demand and Capacity Management
+
+Turbulence in the global economy, technological change, and limited availability of resources are impacting industries. Catena-X enables a secure exchange of demand and capacity data of all partners involved in the automotive network: from raw material to n-tier suppliers to car manufacturers.
```

Comment:
You should mention Eclipse-Tractus-X at least on time in your Readme file, as the open source components are Eclipse-Tractus-X and not Catena-X. Later you can reference Catena-X in the file.

```suggestion
Turbulence in the global economy, technological change, and limited availability of resources are impacting industries. Eclipse-Tractus-X, which hosts the components for a Catena-X network, enables a secure exchange of demand and capacity data of all partners involved in the automotive network: from raw material to n-tier suppliers to car manufacturers.
```